### ISSUE TITLE
Sentinel one is blocking this on mac

### ISSUE THREAD

**Issue by PRanxterr** [bottlepy/bottle#1471]
it's getting flagged and moved to quarantine with the indicator: Malware -File is a Metasploit variant

**Comment by NathanielDayLewis** [comment]
Also getting marked as malware by SentinelOne for me. Attempting to import bottle results in ModuleNotFound error

**Comment by defnull** [comment]
That should be reported to SentinelOne, its an obvious false alarm and there is nothing bottle can do to change that.

**Comment by PRanxterr** [comment]
yes, We did that. Turns out that there are lot of false alarm coming up on SentinelOne

**Comment by defnull** [comment]
Closing as we cannot do anything about bad security products false-flagging bottle as malware :) The issue may also be resolved already, not a single vendor on virustotal detects bottle.py as malware.
